torch/ao/quantization/observer.py

```diff
@@ -1183,6 +1183,9 @@ def forward(self, x_orig: torch.Tensor) -> torch.Tensor:
             assert (
                 combined_min.numel() == 1 and combined_max.numel() == 1
             ), "histogram min/max values must be scalar."
+
+            combined_min, combined_max = combined_min.detach(), combined_max.detach()
```

Comment:
comment?

Reply:
TorchScript causes combined_min, combined_max to requires_grad even though it shouldn't based on upstream tensors.